starcli/__main__.py

```diff
@@ -24,7 +24,14 @@
 
 
 @click.command()
-@click.option("--lang", "-l", type=str, default="", help="Language filter eg: python")
+@click.option(
+    "--lang", 
+    "-l", 
+    type=str, 
+    default="", 
+    help="Language filter eg: python"
```

Comment:
Perhaps have this feature reflected in the help text too? (Like in `--topic`)

Reply:
Get it, I will work on this